### DISCUSSION TITLE
Can this be deployed across multiple Vlans like Deception?

### DISCUSSION THREAD

**Discussion by Jaydeepr-creator** [telekom-security/tpotce#816] [Q&A]
What I am trying to understand is that, if the client has multiple VLAN's and we want to deploy honeypots in each VLAN how do we do that?

**Comment by Ganesh-403** [comment]
@Jaydeepr-creator 
Yes, this is possible, but the recommended approach is **not** to connect a single T-Pot instance directly to every VLAN.

For environments with multiple VLANs or network segments, T-Pot supports a **distributed deployment** consisting of:

- **Hive** – the central node running the Elastic Stack, Web UI, and management services.
- **Sensor(s)** – one or more T-Pot instances that run the honeypots and securely forward their logs to the Hive. :contentReference[oaicite:0]{index=0}

A typical architecture would look like:

```text
                  +----------------------+
                  |      T-Pot Hive      |
                  | ELK + Web UI + Mgmt  |
                  +----------+-----------+
                             ^
                             |
                  Secure log forwarding
                             |
        +--------------------+--------------------+
        |                    |                    |
+---------------+    +---------------+    +---------------+
| Sensor VLAN10 |    | Sensor VLAN20 |    | Sensor VLAN30 |
| Honeypots     |    | Honeypots     |    | Honeypots     |
+---------------+    +---------------+    +---------------+
```

This has several advantages:

- Each VLAN has its own honeypot(s), making the deployment appear more realistic to attackers.
- Traffic remains local to each network segment.
- All events are aggregated centrally in the Hive for analysis and dashboards.
- The architecture scales well as additional VLANs are added. :contentReference[oaicite:1]{index=1}

If you don't want to deploy a Sensor in every VLAN, another option is to place a Sensor where it can receive traffic from multiple VLANs (for example, via SPAN/mirror ports or appropriate routing). However, that depends on your network design and what you're trying to monitor. From what I can tell, the T-Pot documentation doesn't prescribe a specific network topology for this scenario—it focuses on the supported **Hive + Sensor** architecture. :contentReference[oaicite:2]{index=2}

One question that would help provide more specific guidance:

- Are you trying to **attract attacks on each VLAN** (active deception), or are you looking to **monitor traffic** from multiple VLANs using a single T-Pot instance? The recommended deployment differs depending on your objective.

If this solves your problem, feel free to mark it as the accepted answer so others can find it easily.
